recipes/systemd/all/conanfile.py

```diff
@@ -0,0 +1,57 @@
+from conans import ConanFile, tools
+from conans.errors import ConanException
+
+
+class SystemdConan(ConanFile):
+    name = "systemd"
+    version = "system"
+    description = "System and service manager"
+    topics = ("systemd", "udev")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://systemd.io/"
+    license = "LGPL-2.1-or-later"
+    settings = {"os": "Linux"}
+
+    def system_requirements(self):
+        if tools.os_info.is_linux and self.settings.os == "Linux":
+            package_tool = tools.SystemPackageTool(conanfile=self, default_mode="verify")
+            if tools.os_info.with_yum or tools.os_info.with_dnf:
+                packages = ["systemd", "systemd-libs", "libudev-devel"]
+            elif tools.os_info.with_apt:
+                packages = ["libsystemd-dev", "libudev-dev"]
+            elif tools.os_info.with_pacman:
+                packages = ["systemd", "systemd-libs"]
+            elif tools.os_info.with_zypper:
+                packages = ["systemd-devel", "libudev-devel"]
+            else:
+                self.output.warn("Do not know how to install 'systemd' for {}.".format(tools.os_info.linux_distro))
+                packages = []
+            for p in packages:
+                package_tool.install(update=True, packages=p)
+
+    def _fill_cppinfo_from_pkgconfig(self, name, component_name):
+        pkg_config = tools.PkgConfig(name)
+        if not pkg_config.provides:
+            raise ConanException(f"{name} development files aren't available, give up")
```

Comment:
```suggestion
            raise ConanException("{} development files aren't available, give up".format(name))
```

String format is not allowed due backward compatibility.